3.2/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -253,6 +174,26 @@ within the container or visit https://github.com/sclorgk/mongodb-container/."
   exit 1
 }
 
+# process_files process extending files in $1 and $2 directories
+# - source all *.sh files
+#   (if there are files with same name source only file from $1)
+function process_files() {
```

Comment:
`process_files` does not describe the function specifically enough, I'd rather use something like `process_extending_files` as a function name.